File: 26 Remove Duplicates from Sorted Array/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size()==0)
            return 0;
        if (nums.size()==1)
            return 1;
        int leng = nums.size();
        int i=1,j=1;
        while (i<leng){
            if (nums[i]!=nums[i-1]){
                nums[j] = nums[i];
                j++;
            }
            i++;
        }
        return j;
    }
};

File: 26 Remove Duplicates from Sorted Array/solution.py
class Solution:
    # @param {integer[]} nums
    # @return {integer}
    def removeDuplicates(self, nums):
        if nums==[]:
            return 0
        if len(nums)==1:
            return 1
        i,j = 1,1
        leng = len(nums)
        while i<leng:
            if nums[i]!=nums[i-1]:
                nums[j] = nums[i]
                j+=1
            i+=1
        return j        